ietf-layer0-types.yang

```diff
@@ -1563,7 +1570,7 @@ module ietf-layer0-types {
           config false;
           description
             "The set of attributes for an explicit mode";
-          uses common-explicit-mode;
+          // uses explicit-mode;
```

Comment:
better to remove the commented line

Reply:
Look at the comment below https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext-RFC9093-bis/pull/93#discussion_r1497237004 
The commented line is there as placeholder in case the designer has chosen to add the parameters describing explicit-mode, directly here and not by template.